<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Model</class>
 <widget class="QMainWindow" name="Model">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>781</width>
    <height>619</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Model</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="mainLayout">
      <item>
       <widget class="QTabWidget" name="modelTabs">
        <property name="currentIndex">
         <number>0</number>
        </property>
        <widget class="QWidget" name="transactionTab">
         <attribute name="title">
          <string>Transaction</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_4">
          <item>
           <layout class="QVBoxLayout" name="transactionLayout" stretch="0,0,0,0">
            <item>
             <widget class="QTableWidget" name="transactionDataTable">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <attribute name="horizontalHeaderDefaultSectionSize">
               <number>40</number>
              </attribute>
              <row>
               <property name="text">
                <string>00000000</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>00000010</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>00000020</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>00000030</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>00000040</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>00000050</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>00000060</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>00000070</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>00000080</string>
               </property>
              </row>
              <row>
               <property name="text">
                <string>00000090</string>
               </property>
              </row>
              <column>
               <property name="text">
                <string>00</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>01</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>02</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>03</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>04</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>05</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>06</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>07</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>08</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>09</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>0A</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>0B</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>0C</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>0D</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>0E</string>
               </property>
              </column>
              <column>
               <property name="text">
                <string>0F</string>
               </property>
              </column>
              <item row="0" column="0">
               <property name="text">
                <string>30</string>
               </property>
              </item>
              <item row="0" column="1">
               <property name="text">
                <string>30</string>
               </property>
              </item>
              <item row="0" column="2">
               <property name="text">
                <string>30</string>
               </property>
              </item>
              <item row="0" column="3">
               <property name="text">
                <string>32</string>
               </property>
              </item>
              <item row="0" column="4">
               <property name="text">
                <string>30</string>
               </property>
              </item>
              <item row="0" column="5">
               <property name="text">
                <string>31</string>
               </property>
              </item>
              <item row="0" column="6">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="7">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="8">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="9">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="10">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="11">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="12">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="13">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="14">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="0" column="15">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="0">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="1">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="2">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="3">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="4">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="5">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="6">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="7">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="8">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="9">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="10">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="11">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="12">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="13">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="14">
               <property name="text">
                <string/>
               </property>
              </item>
              <item row="1" column="15">
               <property name="text">
                <string/>
               </property>
              </item>
             </widget>
            </item>
            <item>
             <widget class="QLabel" name="tableDataLabel">
              <property name="text">
               <string>Transaction data will be displayed here....</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="transactionValuesLayout" stretch="1,1,1">
              <item>
               <layout class="QVBoxLayout" name="protocolLayout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="methodlabel">
                  <property name="text">
                   <string>Method</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QRadioButton" name="methodRadioButton">
                  <property name="text">
                   <string>From fields</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="startSignLabel">
                  <property name="text">
                   <string>Start Sign</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="startSignComboBox">
                  <property name="editable">
                   <bool>false</bool>
                  </property>
                  <item>
                   <property name="text">
                    <string>0002</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="protocolVersionLabel">
                  <property name="text">
                   <string>Protocol Version</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="protocolVersionComboBox">
                  <item>
                   <property name="text">
                    <string>01</string>
                   </property>
                  </item>
                 </widget>
                </item>
                <item>
                 <widget class="QLabel" name="endSignLabel">
                  <property name="text">
                   <string>End Sign</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="endSignComboBox">
                  <item>
                   <property name="text">
                    <string>0003</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="cardLayout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="panLabel">
                  <property name="text">
                   <string>PAN</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="panLineEdit"/>
                </item>
                <item>
                 <widget class="QLabel" name="nameLabel">
                  <property name="text">
                   <string>Cardholder Name</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="carhholderNameEdit"/>
                </item>
                <item>
                 <widget class="QLabel" name="pinLabel">
                  <property name="text">
                   <string>PIN</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="pinLineEdit"/>
                </item>
                <item>
                 <widget class="QLabel" name="expDateLabel">
                  <property name="text">
                   <string>Expiry Date</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QDateEdit" name="expDateEdit">
                  <property name="displayFormat">
                   <string>MM/yy</string>
                  </property>
                  <property name="date">
                   <date>
                    <year>2020</year>
                    <month>1</month>
                    <day>1</day>
                   </date>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QVBoxLayout" name="textDataLayout">
                <property name="spacing">
                 <number>0</number>
                </property>
                <item>
                 <widget class="QLabel" name="textDataLabel">
                  <property name="text">
                   <string>Text Data</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QPlainTextEdit" name="textDataPlainTextEdit"/>
                </item>
               </layout>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="terminalButtonsLayout">
              <item>
               <widget class="QPushButton" name="testTransactionButton">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>Test</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="saleTransactionButton">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>Sale</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="refundTransactionButton">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>Refund</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="balanceTransactionButton">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>Balance</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="settlementTransactionButton">
                <property name="cursor">
                 <cursorShape>PointingHandCursor</cursorShape>
                </property>
                <property name="text">
                 <string>Settlement</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
        <widget class="QWidget" name="termialTab">
         <attribute name="title">
          <string>Terminal</string>
         </attribute>
         <layout class="QVBoxLayout" name="verticalLayout_10">
          <item>
           <layout class="QVBoxLayout" name="terminalLayout">
            <item>
             <widget class="QLabel" name="terminalLog">
              <property name="text">
               <string>Terminal Log</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QListWidget" name="terminalLogList"/>
            </item>
           </layout>
          </item>
         </layout>
        </widget>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>781</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
